js/filter-user.js

```diff
@@ -0,0 +1,92 @@
+import {removeAllMarkers} from './map.js';
+import {debounce} from './debounce.js';
+
+const OFFER_COUNT = 10;
+const RERENDER_DELAY = 500;
+
+const Price = {
+  middle: 10000,
+  high: 50000,
+};
```

Comment:
```suggestion
const Price = {
  MIDDLE: 10000,
  HIGH: 50000,
};
```
У перечислений ключи должны именоваться как константы, [критерий Б8](https://up.htmlacademy.ru/javascript-individual/1/criteries#b8).